app/frontend/wallet/constants.ts

```diff
@@ -30,6 +30,16 @@ export const NETWORKS: {[key: string]: Network} = {
     epochsToRewardDistribution: 4,
     minimalOutput: 1000000,
   },
+  MARY_TESTNET: {
+    name: 'mary-testnet',
+    networkId: NetworkId.TESTNET,
+    protocolMagic: ProtocolMagic.MARY_TESTNET,
+    eraStartSlot: 4492800, // 21600 slot x 208 epochs
+    eraStartDateTime: Date.parse('29 Jul 2020 21:44:51 UTC'),
```

Comment:
are we sure eraStartSlot and eraStartDateTime  constants are the same for Mary testnet?  If it's not relevant, do we need them anymore?

Reply:
Its in the PR description as TODO :D, I ll fix those.

Reply:
I see, sorry I missed that